src/platform/packages/shared/kbn-unified-data-table/src/components/compare_documents/hooks/calculate_diff.ts

```diff
@@ -38,11 +38,11 @@ export const calculateDiff = ({ diffMode, baseValue, comparisonValue }: Calculat
   }
 
   if (diffMode === 'words') {
-    return diffWords(formattedBaseValue, formattedComparisonValue, { ignoreWhitespace: false });
+    return diffWordsWithSpace(formattedBaseValue, formattedComparisonValue);
   }
 
   return baseValueIsJson && comparisonValueIsJson
-    ? diffJson(formattedBaseValue, formattedComparisonValue, { ignoreWhitespace: false })
+    ? diffJson(formattedBaseValue, formattedComparisonValue)
```

Comment:
`ignoreWhitespace: false` was never supported for `diffJson` and was just a bug in the types, which they fixed with the TS migration. There's no code path for it in the source.